### DISCUSSION TITLE
v4.5.0

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#5314] [Announcements]
v4.5.0

Code v1.68.1

### Changed

- Updated codecov to use codecov uploader
- Moved integration tests to Jest
- Fixed docker release to only download .deb
- Upgraded to Code 1.68.1
- Install `nfpm` from GitHub
- Upgraded to TypeScript 4.6

### Added

- Added tests for `open`, `isWsl`, `handlePasswordValidation`
- Provided alternate image registry to dockerhub
- Allowed users to have scripts run on container with `ENTRYPOINTD` environment
  variable

### Fixed

- Fixed open CLI command to work on macOS

<hr /><em>This discussion was created from the release <a href='https://github.com/coder/code-server/releases/tag/v4.5.0'>v4.5.0</a>.</em>

**Comment by jsjoeio** [comment]
Heads-up: looks like GitHub Packages might be having issues https://github.com/coder/code-server/runs/7156893600?check_suite_focus=true#step:9:1351

<img width="1712" alt="image" src="https://user-images.githubusercontent.com/3806031/176969567-da0b3419-8d5e-457b-9f89-8c562f538b68.png">

**Reply by onedr0p** [reply]
I just came here to mention this. I also cannot pull the image:

```
❯ docker pull ghcr.io/coder/code-server:v4.5.0
Error response from daemon: manifest unknown
```

**Comment by onedr0p** [comment]
@jsjoeio are you sure packages are set up correctly on the coder org? I see this in the build log:

```
#30 ERROR: failed commit on ref "manifest-sha256:8253f06a3a2c0498beceb9c5962f58b2b03786b50fdcf912a24bbf94065e9d08": unexpected status: 403 Forbidden
```

**Reply by jsjoeio** [reply]
I think so. Does 4.5.1 work for you? I see it [here](https://github.com/coder/code-server/pkgs/container/code-server)

**Reply by onedr0p** [reply]
Yep, all good now. Thanks!
